webpack.dev.config.ts

```diff
@@ -51,6 +51,10 @@ module.exports = {
           }
         }
       ]
+    },
+    {
+      test: /\.html$/,
+      loader: 'underscore-template-loader'
```

Comment:
Add these also to _webpack.prod.config.ts_ and maybe _karma.config.ts_

Reply:
Okay